server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/variables/VariableType.java

```diff
@@ -25,5 +25,6 @@
   MULTI_TABLE_DATA_NAME,
   RENAME_ITEM,
   TABLE_DATA_NAME,
-  MAP_NAME
+  MAP_NAME,
+  FD_SD
```

Comment:
Would it make sense to separate them for the purpose of check for the effective type?

Reply:
Yes, I would work on it